apps/organizations/tests/test_org_models.py

```diff
@@ -157,17 +158,33 @@ def test_slug_validated_against_denylist(self):
         denied_word = DenyListWordFactory()
         rp = RevenueProgram(name="My rp", organization=self.organization, payment_provider=self.payment_provider)
         rp.slug = denied_word.word
-        with self.assertRaises(ValidationError) as validation_error:
+        with pytest.raises(ValidationError) as validation_error:
             rp.clean_fields()
-        self.assertIn("slug", validation_error.exception.error_dict)
-        self.assertEqual(SLUG_DENIED_CODE, validation_error.exception.error_dict["slug"][0].code)
-        self.assertEqual(GENERIC_SLUG_DENIED_MSG, validation_error.exception.error_dict["slug"][0].message)
+        assert "slug" in validation_error.value.error_dict
+        assert SLUG_DENIED_CODE == validation_error.value.error_dict["slug"][0].code
+        assert GENERIC_SLUG_DENIED_MSG == validation_error.value.error_dict["slug"][0].message
 
     def test_admin_benefit_options(self):
-        self.assertTrue(isinstance(self.instance.admin_benefit_options, list))
+        assert isinstance(self.instance.admin_benefit_options, list)
 
     def test_admin_benefitlevel_options(self):
-        self.assertTrue(isinstance(self.instance.admin_benefitlevel_options, list))
+        assert isinstance(self.instance.admin_benefitlevel_options, list)
+
+    @pytest.mark.django_db
+    @pytest.mark.parametrize("fiscal_sponsor_name, expected", (("", False), (None, False), ("NRH", True)))
```

Comment:
This is nitpicking, but i've been omitting space between commas in first arg to `.parametrize`. Though spaces are fine, i just followed convention in pytest docs (no space), and would prefer to keep this consistent throughout.  If there's a strong argument for adding space, that's something we can consider too, but probably would want to handle in a separate ticket. 

```suggestion
    @pytest.mark.parametrize("fiscal_sponsor_name,expected", (("", False), (None, False), ("NRH", True)))
```